src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -1264,6 +1310,13 @@ private static void checkUserRecord(UserRecord userRecord) {
     assertEquals("gold", claims.get("package"));
   }
 
+  private static void checkTenant(Tenant tenant) {
```

Comment:
Suggestion: Just to be in safe side, let's give your test tenants different IDs (say TENANT_ID_1 and TENANT_ID_2). Then change the signature of this method to accept both `Tenant` and `String tenantId`, so the caller can pass them in to verify.

Reply:
Done.